### ISSUE TITLE
ts file which has declarations only will not emit corresponding d.ts

### ISSUE THREAD

**Issue by littlee** [developit/microbundle#868]
src:
index.ts
```ts
import { T } from './model';
export function a(b: T) {}
```
model.ts
```ts
export type T = {
  a: string;
}
```
package.json
```json
{
  "name": "my-lib",
  "version": "1.0.0",
  "types": "types/index.d.ts",
  "scripts": {
    "build": "microbundle"
  },
  "devDependencies": {
    "microbundle": "^0.13.3"
  }
}
```

after building, `./types` directory has only one file, `model.d.ts` is missing
![image](https://user-images.githubusercontent.com/7534668/126439084-bca34d9c-6be0-43c8-b651-e28049bc2da6.png)

**Comment by rschristian** [comment]
@littlee This happens because you (likely) haven't set your `tsconfig.json` up correctly. You need to make sure both files are included by your config, for example:

```json
{
  ...
  "include": ["src/**/*"]
}
```

This ensures `types/model.d.ts` gets created. See https://github.com/ezolenko/rollup-plugin-typescript2/issues/106#issuecomment-415461839 for some more information.

**Comment by littlee** [comment]
🤦  alright

**Comment by agilgur5** [comment]
Thought I'd update here that I fixed the root cause in https://github.com/ezolenko/rollup-plugin-typescript2/pull/406, which was released in rpt2 [`0.34.0`](https://github.com/ezolenko/rollup-plugin-typescript2/releases/tag/0.34.0)
